test/monitor.js

```diff
@@ -40,6 +42,13 @@ const it = lab.it;
 
 describe('Monitor', () => {
 
+    // Each test run can potentally add listeners to wreck if monitor.stop() is not called
+    lab.beforeEach((done) => {
+
+        Wreck.removeAllListeners();
+        done();
+    });
+
```

Comment:
Previously (see #455) this was accomplished by calling Wreck.setMaxListeners(0). I thought this was a cleaner solution and allows me to test the listener is removed in monitor.stop() (see below). Is this ok?